networkx/algorithms/flow/__init__.py

```diff
@@ -1,12 +1,6 @@
-from .maxflow import *
-from .mincost import *
-from .edmondskarp import *
-from .preflowpush import *
-from .shortestaugmentingpath import *
-from .capacityscaling import *
-from .networksimplex import *
-from .utils import build_flow_dict, build_residual_network
-
+# this import is used to combine the __all__ varibles.
```

Comment:
The effect of ```__all__``` occurs when importing the flow module.
```from flow import *```
imports the items listed in the ```__all__``` variable defined in ```flow/__init__.py```
So we need an __all__ variable for the ```flow``` module (or maybe flow is a package...)

Anyway, sphinx doesn't build the docs correctly without it.